site/cypress/e2e/Cypress-Feature/polls.spec.js

```diff
@@ -260,6 +271,24 @@ describe('Test cases revolving around polls functionality', () => {
         cy.login();
         cy.visit(['sample', 'polls']);
         cy.contains('Poll Cypress Test').siblings(':nth-child(6)').children().click();
+        cy.wait(5000); // Waiting for duration to reach 0, so poll ends.
+        cy.reload(); // Will not need this after websockets.
+        cy.contains('Poll Cypress Test').siblings(':nth-child(6)').children().should('not.be.checked');
+        //Removing duration to continue testing
+        //Editing the poll to remove timer
+        cy.contains('Poll Cypress Test').siblings(':nth-child(1)').children().click();
+        //Checking if user input for duration saved
+        cy.get('#enable-timer').should('be.checked');
+        cy.get('#timer-inputs').get('#poll-hours').invoke('val').should('eq', '0');
+        cy.get('#timer-inputs').get('#poll-minutes').invoke('val').should('eq', '0');
+        cy.get('#timer-inputs').get('#poll-seconds').invoke('val').should('eq', '5');
+        cy.get('#poll-seconds').clear();
+        cy.get('#enable-timer').get('#poll-hours').clear().type('3');
+        cy.get('button[type=submit]').click();
```

Comment:
Is there a more specific selector to target this button with?  This will cause problems if another form with a submit button is added to the page in the future.

Reply:
This is the same way as the pre-existing logic for editing a poll and submitting changes.